examples/templates/node/tsconfig.json

```diff
@@ -1,8 +1,8 @@
 {
     "compilerOptions": {
         "strict": true,
-        "moduleResolution": "node16",
-        "module": "NodeNext",
-        "esModuleInterop": true        
+        "moduleResolution": "node",
```

Comment:
I believe the oldest combat setup is:
```
"module": "ES2022",
"moduleResolution": "node16",
```

Was there any issue so we had to increase it @afmika?

Reply:
Oh, my bad. I didn't mean to commit that. I was messing around with that because the old `load-tgraph` (which you can see I removed in `examples/templates/node/packages.json`) wouldn't work for some reason I and I was trying to fix that.